src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/Methodologies/MethodologyServiceTests.cs

```diff
@@ -161,6 +164,73 @@ await using (var context = InMemoryApplicationDbContext(contentDbContextId))
             }
         }
 
+        [Fact]
+        public async Task GetUnpublishedReleasesUsingMethodology_PublicationsHaveNoUnpublishedReleases()
```

Comment:
I was also expecting to see a test for when there *are* unpublished Releases, unless it's not necessary for a reason?

Reply:
These tests did exist but I accidently lost them with a merge today! 😱 I've reinstated them now, thanks.